app/services/send_applications_to_provider.rb

```diff
@@ -1,4 +1,4 @@
-# This service class will be called as a background job, scheduled to run nightly
+# This worker will be scheduled to run nightly
```

Comment:
This comment is now out of date as it runs every hour 🙂

Reply:
in theory each hourly run except the first one after midnight should be a no-op. The extra runs give us a bit of a safety net in case of Redis or Sidekiq being down.